### DISCUSSION TITLE
Improvement+

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#590] [Q&A] [answered]
I've noticed that if you research in a turn where you have +1 research, at the bottom it shows "R&D x% IMPROVEMENT+".  That plus sign must represent the bonus to research, and must have been intentional - but I haven't heard anyone mention it before.  What was the rationale for that?

![Improvement+](https://user-images.githubusercontent.com/3118014/136644386-8622d186-cfcd-4f31-84b3-9bd88b8460a9.png)

**Comment by rnyoakum** [comment, marked as answer]
I looked into the history of the '+' sign. It's present in the initial commit in the repository, so I checked the github [mirror](https://github.com/raceintospace/raceintospace-cvs) of the old CVS repository. As far back as the [initial revision](https://github.com/raceintospace/raceintospace-cvs/commit/be837a6c332262957b4cb73edf8f0614a7d92624) there is code to add the plus sign.

```
    PrintAt(0,0,"% IMPROVEMENT");
    if (Data->P[plr].RDMods>0) PrintAt(0,0,"+");
``` 

So it appears the '+' sign has been around since 2007.

**Reply by peyre** [reply]
HAHAHA!!!  Well, that shows me up, all right!  How did I miss that?  Thanks Ryan.

Come to think of it, I guess I could've looked that one up for myself.  Oops.  So, do you think it's good to have it, or would it be better removed?
